### ISSUE TITLE
Nmap UDP scan error persistence 

### ISSUE THREAD

**Issue by rfare** [1N3/Sn1per#37]
Installed on latest Kali Rolling as of writing this [4.6.2-2kali2 (2016-06-28)], installation went fine, and all functions well aside from 2 issues:
Jexboss fails to run, citing an unknown argument. This was easily fixed in the script, by adding the "-host" argument.
The second is I cannot get nmap to do a UDP scan. Every attempt get's me the same error:
```
Starting Nmap 7.12 ( https://nmap.org ) at 2016-07-20 00:53 EDT
WARNING: a TCP scan type was requested, but no tcp ports were specified.  Skipping this scan type.
Nmap done: 1 IP address (1 host up) scanned in 0.13 seconds
```
I've changed the line to
`nmap -sU -T5 --open -p U:53,U:67,U:68,U:88,U:161,U:162,U:137,U:138,U:139,U:389,U:520,U:2049 $TARGET`
and
`nmap -sU -T5 --open -p 53,67,68,88,161,162,137,138,139,389,520,2049 $TARGET`
Even went so far as to add a quick, hackish, self ping timeout between the TCP and UDP scans, but no dice. All 3 lines run fine outside of the script. I'm totally stumped as to a working solution.

**Comment by 1N3** [comment]
Hmm.. interesting. They must have changed some things in Kali rolling that are causing this. I'll try to install it at some point to test and see if I figure out what's happening. Thanks for the heads up.

**Comment by rfare** [comment]
Not a problem. I'll keep digging for working solutions as well. Hopefully one of us will figure it out eventually.

**Comment by windmillnl** [comment]
Same error here on Kali :(
Couldn't fix it either, did you guys ever figure it out?

**Comment by 1N3** [comment]
Still looking into this. Will keep you posted.

**Comment by windmillnl** [comment]
Thank you 1N3, great work btw! Love this script!

**Comment by windmillnl** [comment]
Actually, I've encountered another thing:
When the scan is done it's generating the report but it never makes it to the folder.
No errors or anything but the folders are always empty.

**Comment by rfare** [comment]
Found the solution, the -sU flag is missing on all 4 Nmap UDP scans in the script.
I clearly remember it being optional back in Nmap 5/6, as long as the ports were prefixed with U: (Like you had them). Not sure when that change was introduced O.o

**Comment by 1N3** [comment]
@rfare thanks for your help! This is fixed in 1.8b.
